frontend/src/scenes/funnels/funnelUtils.ts

```diff
@@ -348,16 +348,23 @@ export const getClampedStepRangeFilter = ({
     filters: FilterType
 }): FunnelStepRangeEntityFilter => {
     const maxStepIndex = Math.max((filters.events?.length || 0) + (filters.actions?.length || 0) - 1, 1)
-    const funnel_from_step = clamp(stepRange?.funnel_from_step ?? filters.funnel_from_step ?? 0, 0, maxStepIndex)
-    return {
-        ...(stepRange as FunnelStepRangeEntityFilter),
-        funnel_from_step,
-        funnel_to_step: clamp(
-            stepRange?.funnel_to_step ?? filters.funnel_to_step ?? maxStepIndex,
-            funnel_from_step + 1,
-            maxStepIndex
-        ),
+
+    const hasFunnelFromStep = typeof (stepRange?.funnel_from_step || filters.funnel_from_step) === 'number'
+    const hasFunnelToStep = typeof (stepRange?.funnel_to_step || filters.funnel_to_step) === 'number'
+
+    if (hasFunnelFromStep || hasFunnelToStep) {
+        const funnel_from_step = clamp(stepRange?.funnel_from_step ?? filters.funnel_from_step ?? 0, 0, maxStepIndex)
+        return {
+            ...(stepRange as FunnelStepRangeEntityFilter),
+            funnel_from_step,
+            funnel_to_step: clamp(
+                stepRange?.funnel_to_step ?? filters.funnel_to_step ?? maxStepIndex,
+                funnel_from_step + 1,
+                maxStepIndex
+            ),
+        }
     }
+    return {}
```

Comment:
I think differentiating between having the steps set and unset will cause some funky behavior here. To the user there is no difference when a funnel step is set to 0 vs when it's set to 0 by default.

Reply:
Good call on this. Fixed it as part of the bug below!